kvbc/src/ReplicaImp.cpp

```diff
@@ -187,6 +187,16 @@ Status ReplicaImp::addBlock(const SetOfKeyValuePairs &updates, BlockId &outBlock
 
 void ReplicaImp::set_command_handler(ICommandsHandler *handler) { m_cmdHandler = handler; }
 
+void ReplicaImp::set_app_state(std::shared_ptr<bftEngine::SimpleBlockchainStateTransfer::IAppState> appState) {
```

Comment:
Does it make sense to create another method for the case of a `nullptr` parameter being passed? Maybe something like `set_default_app_state()` or similar. If we do that, then `set_app_state()` might be just a simple setter.